odyssey-app/src/main/java/com/codebutler/odyssey/app/feature/common/FpsCalculator.kt

```diff
@@ -0,0 +1,27 @@
+package com.codebutler.odyssey.app.feature.common
```

Comment:
Please move all these new classes into `com.codebutler.odyssey.lib.rendering` instead of the app module.

Reply:
👍